src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -269,29 +265,29 @@ function MoneyTemporaryForRefactorRequestParticipantsSelector({participants, onF
     const headerMessage = useMemo(
         () =>
             OptionsListUtils.getHeaderMessage(
-                lodashGet(newChatOptions, 'personalDetails', []).length + lodashGet(newChatOptions, 'recentReports', []).length !== 0,
-                Boolean(newChatOptions.userToInvite),
+                ((newChatOptions as Options)?.personalDetails ?? []).length + ((newChatOptions as Options)?.recentReports ?? []).length !== 0,
+                Boolean((newChatOptions as Options)?.userToInvite),
```

Comment:
```suggestion
                !!(newChatOptions as Options)?.userToInvite,
```

Reply:
I think it better to leave straight-type conversion here since it is less complicated to understand